README.md

```diff
@@ -57,6 +57,7 @@ const options = {
     '/healthcheck': healthCheck,    // a function returning a promise indicating service health,
     verbatim: true // [optional = false] use object returned from /healthcheck verbatim in response
   },
+  caseInsensitive, // [optional] whether given health checks routes are case insensitive (defaults to false) 
```

Comment:
how about making this the default behaviour?

Reply:
@gergeltke I would prefer that too, but bear in mind, it's a breaking change since you can have a case sensitive route in express (or koa) (yes, very rarely done), and now you can override that route with the healthcheck.

If you don't mind about this very extremely unlikely scenario, I would be happy to make it the default.

Reply:
That makes sense - let's err on the side of being bullet-proof, and making it opt-in. Thank you for the contribution Doron!